### PR TITLE
Emily autocomplete

### PR DESCRIPTION
**Pull request by mgunnhawkins**
All AC criteria have been met...future bug fix not currently in story is that the directions still show up as null.

### CODE REVIEW

**Comment by krystalwhite** [comment]
Hi, Melanie. There are multiple lines of code that are commented out in this pull request. Would you please either reactive those pieces if they are supposed to be active or delete them if they are not needed?

**Comment by mgunnhawkins** [comment]
> Hi, Melanie. There are multiple lines of code that are commented out in this pull request. Would you please either reactive those pieces if they are supposed to be active or delete them if they are not needed?

Code updated!

File: src/main/resources/templates/index.html [hunk 1]
```diff
@@ -2,53 +2,51 @@
...
+                            <label>Name Your Trip:
+                                <input class="input" type="text" th:field="${trip.tripName}" th:value="${trip.tripName}">
+                            </label>
+                            <br>
+                            <p class="error" th:errors="${trip.tripName}"></p>
+                            <br>
+                            <label th:for="startingLocation">Starting Location:
+                                <input class="input directionInput" type="text" id="originInput" placeholder="starting from...">
+                            </label>
+                    <!--        th:field="${trip.startingLocation}" th:value="${trip.startingLocation}" for above-->
```
Comment (emmykg38) [inline src/main/resources/templates/index.html]: line 21 is a necessary piece, please add back in!

File: src/main/resources/static/script.js [hunk 2]
```diff
@@ -1,10 +1,41 @@
...
+     componentRestrictions: {'country': ['us']},
+     fields: ['geometry', 'name', 'formatted_address']
+   });
+   var destinationInput = document.getElementById("destinationInput");
+   var destination = new google.maps.places.Autocomplete(destinationInput,
+   {
+    componentRestrictions: {'country': ['us']},
+    fields: ['geometry', 'name', 'formatted_address'] //is there anything else we need? we'll want to create a location object with this info
+   });
+   //handle when a user hits enter upon finding selected place
```
Comment (emmykg38) [inline src/main/resources/static/script.js]: can we remove unnecessary comments from this file? line 8, 19, 21

File: src/main/java/com/roadtriprunner/RoadTripRunner/models/AbstractEntity.java [hunk 3]
```diff
@@ -1,6 +1,6 @@
 
 package com.roadtriprunner.RoadTripRunner.models;
-import lombok.Data;
+//import lombok.Data;
```
Comment (emmykg38) [inline src/main/java/com/roadtriprunner/RoadTripRunner/models/AbstractEntity.java]: can we reinstate this or delete if unnecessary?

**Review by emmykg38 — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/index.html (hunk 1 above)
Comment (mgunnhawkins) [reply src/main/resources/templates/index.html]: resolved

File: src/main/resources/static/script.js (hunk 2 above)
Comment (mgunnhawkins) [reply src/main/resources/static/script.js]: resolved

File: src/main/java/com/roadtriprunner/RoadTripRunner/models/AbstractEntity.java (hunk 3 above)
Comment (mgunnhawkins) [reply src/main/java/com/roadtriprunner/RoadTripRunner/models/AbstractEntity.java]: resolved

File: src/main/resources/static/script.js [hunk 4]
```diff
@@ -1,10 +1,39 @@
...
+              event.preventDefault();
+              }
+            });
+       let endInput = document.getElementById("destinationInput");
+              google.maps.event.addDomListener(destinationInput, "keydown", function(event) {
+              if (event.keyCode === 13){
+                event.preventDefault();
+                }
+              });
+
```
Comment (emmykg38) [inline src/main/resources/static/script.js:34]: I have one more request! The script.js file is currently styled with two spaces per indentation, and it looks like this new code is uses 4 spaces/a full tab. can you refactor to keep the indentation congruent throughout? TYSM!

Comment (mgunnhawkins) [reply src/main/resources/static/script.js:34]:
```suggestion
changed to 2 spaces throughout
```

**Review by emmykg38 — APPROVED** [review]

**Review by krystalwhite — APPROVED** [review]

**Pull request merged**